dietpi/dietpi-software

```diff
@@ -1076,6 +1076,16 @@ _EOF_
 					  aSOFTWARE_TYPE[$index_current]=0
 			 aSOFTWARE_ONLINEDOC_URL[$index_current]='f=8&t=5&p=7575#p7575'
 
+		#------------------
+		index_current=155
```

Comment:
This is the index assigned to the software title, it must be used for all entries of:
```INSTALLING_INDEX=```